ddtrace/internal/constants.py

```diff
@@ -8,14 +8,16 @@
 PROPAGATION_STYLE_B3_MULTI = "b3multi"
 PROPAGATION_STYLE_B3_SINGLE = "b3"
 _PROPAGATION_STYLE_W3C_TRACECONTEXT = "tracecontext"
+_PROPAGATION_STYLE_AWS_XRAY = "xray"
 _PROPAGATION_STYLE_NONE = "none"
-_PROPAGATION_STYLE_DEFAULT = "tracecontext,datadog"
+_PROPAGATION_STYLE_DEFAULT = "tracecontext,datadog,xray"
 PROPAGATION_STYLE_ALL = (
     _PROPAGATION_STYLE_W3C_TRACECONTEXT,
     PROPAGATION_STYLE_DATADOG,
     PROPAGATION_STYLE_B3_MULTI,
     PROPAGATION_STYLE_B3_SINGLE,
     _PROPAGATION_STYLE_NONE,
+    _PROPAGATION_STYLE_AWS_XRAY,
```

Comment:
I'm struggling with the idea of adding XRAY as a propagation style here... Can you please add some explanation of how the propagation is used in the descriptions? Thank you 🙇

Reply:
Will add to the description. But basically  `dd-trace-java` propagates all context for AWS integrations via XRay headers. This work was done in [this PR](https://github.com/DataDog/dd-trace-java/pull/3104). In order for a python consumer to extract context from a message created by a Java producer, we need to add this new propagator.  The other way around ( a python producer to a java consumer) already works since the java tracer tries to extract via both the XRay header, and if that isn't present, then the message's attributes, which is how we currently propagate in the Python tracer.